src/promptflow/tests/test_configs/flows/assistant_tool/flow.dag.yaml

```diff
@@ -0,0 +1,53 @@
+inputs:
+  content:
+    type: string
+    default: I'm going to run for 2 hours today, how much calories will I burn?
+  assistant_id:
+    type: string
+    default: asst_yWhdFYoCS1UatnRRQZGY85aL
+outputs:
+  assistant_output:
+    type: string
+    reference: ${assistant_play.output}
+nodes:
+- name: assistant_play
+  type: python
+  source:
+    type: code
+    path: oai_assistant.py
+  inputs:
+    conn: openai_connection
+    content: ${inputs.content}
+    assistant_id: ${inputs.assistant_id}
+    thread_id: dummy_thread_id
+    assistant_definition:
+      module: gpt-4-1106-preview
+      instructions: You are a helpful assistant.
+      tools:
+      - type: promptflow_tool
```

Comment:
promptflow_tool is this still needed?

Reply:
Yes, need it to avoid mix-ups with the openai tools.

Reply:
Changed promptflow_tool to function.